### PR TITLE
intro wording

### PR DESCRIPTION
**Pull request by testbird**
~Also assuming `action` is an alias for `route`, and using `action`, because the classes' default route is used.~

### CODE REVIEW

**Comment by paulcsmith** [comment]
I don't quite have time to look at the full thing and review it yet, but `action` has been removed so `route` is the way to define a RESTful route

Thanks for working on making the guides better!

**Comment by testbird** [comment]
ok got rid of `action`

**Comment by testbird** [comment]
I have no idea why there is a build error shown.

File: src/actions/guides/http_and_routing/request_and_response.cr [hunk 1]
```diff
@@ -58,7 +58,7 @@ class Guides::HttpAndRouting::RequestAndResponse < GuideAction
     #{permalink(ANCHOR_HANDLING_RESPONSES)}
     ## Handling Responses
 
-    Once you've recieved your request, and handled it, you'll need to return a response. Every Lucky::Action requires that a response is returned.
+    Finally, every action is required to return one of the available responses:
```
Comment (jwoertink) [inline src/actions/guides/http_and_routing/request_and_response.cr:61]: This definitely reads easier 😂

**Review by jwoertink — APPROVED** [review]

File: src/actions/guides/http_and_routing/request_and_response.cr [hunk 2]
```diff
@@ -11,18 +11,18 @@ class Guides::HttpAndRouting::RequestAndResponse < GuideAction
     <<-MD
     ## Handling Requests
 
-    When a request comes in to an Action, Lucky gives you access to the request object through the `request` method.
+    When a request for a path triggers an action, the action has access to the request object through a `request` method.
```
Comment (paulcsmith) [inline src/actions/guides/http_and_routing/request_and_response.cr]: Maybe "calls an action"?

File: src/actions/guides/http_and_routing/request_and_response.cr [hunk 3]
```diff
@@ -11,18 +11,18 @@ class Guides::HttpAndRouting::RequestAndResponse < GuideAction
...
     > The `request` object is an instance of [HTTP::Request](https://crystal-lang.org/api/HTTP/Request.html).
 
-    Lucky gives you access to a few helpful methods based on the request Content-Type.
+    Lucky also provides access to some helpful methods to determine the requested Content-Type.
```
Comment (paulcsmith) [inline src/actions/guides/http_and_routing/request_and_response.cr:17]: Nice! Much better

File: src/actions/guides/http_and_routing/request_and_response.cr [hunk 4]
```diff
@@ -11,18 +11,18 @@ class Guides::HttpAndRouting::RequestAndResponse < GuideAction
...
 
     * `json?` - true if the Content-Type header is "application/json"
     * `ajax?` - true if the X-Requested-With header is "XMLHttpRequest"
     * `html?` - true if the Content-Type header is "text/html"
     * `xml?` - true if the Content-Type header is "application/xml" or "application/xhtml+xml"
     * `plain?` - true if the Content-Type header is "text/plain"
 
-    You can use these methods to help direct the request or return different responses.
+    You can use these methods to direct the request or return different responses.
```
Comment (paulcsmith) [inline src/actions/guides/http_and_routing/request_and_response.cr:25]: Good change

**Review by paulcsmith — COMMENTED** [review]
A couple minor suggestions. Thanks for clarifying this!

**Comment by testbird** [comment]
Ok, hopefully the three PRs are now ok.

**Comment by paulcsmith** [comment]
Thank you so much @testbird!

**Pull request merged**